scripts/valhalla_build_config

```diff
@@ -403,6 +404,7 @@ help_text = {
     'isochrone': {
       'max_contours': 'Maximum number of input contours to allow',
       'max_time': 'Maximum time value for any one contour',
+      'max_distance':'Maximum b-line distance between all locations in meters',
```

Comment:
That's how all of the other descriptions of max_distance are described so I just continued the pattern.